src/mail-app/mail/view/MailGuiUtils.ts

```diff
@@ -96,18 +99,56 @@ async function reportMails(
 	return true
 }
 
+async function undoMoveMails(
```

Comment:
I would rename it to `showUndoSnackbar()` otherwise from the call site it looks like we are actually running the undo